### PR TITLE
feat(invitations): Import e-mail participants from CSV 🗒️ 

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Ref #13451
* On top of API #13871

### 🖼️ Screenshots / Screencasts

🏡 After | 🏡 After
-- | --
image | image
image | --
image | image

### 🚧 Tasks

- [ ] Show invited e-mails in the list
- [ ] Another place for 'Import' button?

### 🏁 Checklist

- [ ] 🌏 Tested with different browsers / clients:
  - [x] Chromium (Chrome / Edge / Opera / Brave)
  - [ ] Firefox
  - [ ] Safari
  - [ ] Talk Desktop
  - [ ] Not risky to browser differences / client
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not requiredSigned-off-by: Maksim Sukharev <antreesy.web@gmail.com>

### CODE REVIEW

File: src/components/ImportEmailsDialog.vue [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+					{{ n('spreed', '%n invitation can be sent', '%n invitations can be sent', uploadResult.invites) }}
+				</p>
+			</template>
+		</div>
+
+		<template #actions>
+			<NcButton type="primary"
+				:disabled="!uploadResult"
+				@click="submitList(importedFile)">
+				{{ t('spreed', 'Submit list from file') }}
```
[the remark this answers is not in the document]
Comment (nickvergessen) [reply src/components/ImportEmailsDialog.vue:158]: I would still call it "Send invites" or something?
cc @DorraJaouad what do you think, as Maksim is on vacation

File: src/components/ImportEmailsDialog.vue [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+ * @param file file to upload
+ */
+async function submitList(file: File) {
+	if (!file) {
+		return
+	}
+
+	try {
+		await importEmails(props.token, file, false)
+		showSuccess(t('spreed', 'Participants added successfully'))
```
Comment (nickvergessen) [inline src/components/ImportEmailsDialog.vue:90]: this should also close the dialog
grafik

Comment (DorraJaouad) [reply src/components/ImportEmailsDialog.vue:87]: fixed

**Comment by DorraJaouad** [comment]
Buttons location:
Why was it related to lobby? IMO it's better if it is in Moderation section under guest access. 

User understanding:
There is no clue on how the file should be (format, structure), we discussed these options:
- Sample file
- Add documentation and add a link in the dialog for that section

**Review by DorraJaouad — APPROVED** [review]

**Pull request merged**